README.md

```diff
@@ -1,2 +1,39 @@
-# Web3Studio JS Standards
+# Web3Studio ESlint Config
+
 A collection of boilerplate config to be used by Web3Studio javascript based projects.
+
+## Installation
+
+```bash
+$ yarn add --dev eslint-config-web3studio eslint prettier pretty-quick husky
+```
+
+In your `.eslintrc.js` file:
+
+```js
+module.exports = {
+  extends: ['web3studio']
+};
+```
+
+In your `.prettierrc.js` file:
+
+```js
+module.exports = require('eslint-config-web3studio/prettier');
+```
+
+In your `package.json` file:
+
+```json
+{
+  "husky": {
+    "hooks": {
+      "pre-commit": "pretty-quick --staged"
+    }
+  },
+  "scripts": {
+    "lint": "eslint .",
+    "prettier": "prettier \"**/*.{js,json,css,md}\" --write"
+  }
+}
```

Comment:
Anyone using this package, (ie. us in the future) will have to also download prettier, husky, etc ourselves correct? Is this worth putting in the README?

Also - if when we use it we require prettier.js and index.js from this package what is the purpose of the .prettierrc and .eslintrc files? Unless there is going to be other code in this repo that requires prettier and linting? Or is it for the sake of the package.json?

Reply:
> Anyone using this package, (ie. us in the future) will have to also download prettier, husky, etc ourselves correct? Is this worth putting in the README?

I believe I did? I included install instructions and file contents, what else am I missing?

> what is the purpose of the .prettierrc and .eslintrc files?

Prettier does a _whole_ lot, it lints markdown, json, js, and a bunch of other languages too. It's actually linting/formatting it's own configuration! As an example, it keeps me from using double quotes in those files.